File: 10_generics/javatests/src/test/java/javatests/MyTuple.java
package javatests;

class MyTuple<T> {

    // I don't recommend doing this in real code
    public static String globalStaticField;

    private final T first;

    private final T second;

    public MyTuple(T first, T second) {
        this.first = first;
        this.second = second;
    }

    public T getFirst() {
        return first;
    }

    public T getSecond() {
        return second;
    }
}

File: 10_generics/dotnet/tests/MyTuple.cs
namespace tests
{
    public class MyTuple<T>
    {
        public static string staticVal;

        public readonly T first;
        public readonly T second;

        public MyTuple(T _first, T _second)
        {
            this.first = _first;
            this.second = _second;
        }
    }
}